common/views/components/WorkHeader/WorkHeader.js

```diff
@@ -60,8 +61,7 @@ const WorkHeader = ({ work }: Props) => {
         {(work.contributors.length > 0 || productionDates.length > 0) && (
           <div
             className={classNames({
-              flex: true,
-              'flex--wrap': true,
+              'flex flex--wrap flex--v-center': true,
```

Comment:
Does this make it harder to refactor later?

Reply:
Not sure – my thinking was any time we wanted the `flex` class we'd also want the `flex--wrap` class, to having them both controllable by the same boolean felt good.